.gitignore

```diff
@@ -5,10 +5,11 @@
 /.vscode/
 /htmlcov/
 /logs
-/mails/
+mails/
```

Comment:
why remove the leading slash?
this would cause any directory named `mails` in the whole project to be ignored.
including any module with that name. when actually we only need to ignore the root mail directory.